geode-redis/src/main/java/org/apache/geode/redis/internal/executor/hash/HExistsExecutor.java

```diff
@@ -53,12 +54,7 @@ public void executeCommand(Command command, ExecutionHandlerContext context) {
     RedisHash map = getRedisHash(context, key);
     boolean hasField = map.containsKey(field);
 
-    if (hasField) {
-      command.setResponse(Coder.getIntegerResponse(context.getByteBufAllocator(), EXISTS));
-    } else {
-      command.setResponse(Coder.getIntegerResponse(context.getByteBufAllocator(), NOT_EXISTS));
-    }
-
+    return RedisResponse.integer(hasField ? EXISTS : NOT_EXISTS);
```

Comment:
What do you think of supporting a flavor of "integer(boolean)" that would respond with 1 if true and 0 if false? I've seen a large number of redis commands that do this

Reply:
Yes, good idea.